.github/workflows/static.yml

```diff
@@ -0,0 +1,22 @@
+on: [push, pull_request]
+
+name: Static analysis
+
+jobs:
+  php-cs-fixer:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: PHP-CS-Fixer
+        uses: docker://jakzal/phpqa:php7.3-alpine
+        with:
+            args: php-cs-fixer fix --dry-run --diff-format udiff -vvv
+
+  phpstan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: PHPStan
+        uses: docker://oskarstark/phpstan-ga
```

Comment:
Build is failing with `jakzal/phpqa`. See https://github.com/php-translation/extractor/pull/139/checks?check_run_id=338018152
@rvanlaak I guess it's the same problem you had.

Reply:
I dont see the build failing..

Reply:
It looks like it's not possible to see a previous build. :thinking: 
It was failing with `jakzal/phpqa` but it's OK with `oskarstark/phpstan-ga`

Reply:
cc @jakzal